apps/api/src/app/events/e2e/bridge-trigger.e2e.ts

```diff
@@ -47,7 +48,7 @@ contexts.forEach((context: Context) => {
       session = new UserSession();
       await session.initialize();
       subscriberService = new SubscribersService(session.organization._id, session.environment._id);
-      subscriber = await subscriberService.createSubscriber();
+      subscriber = await subscriberService.createSubscriber({ _id: session.subscriberId });
```

Comment:
Needed to ensure the subscriber created has the same subscriber Id as the session, so that preferences can be attached to the same subscriber used for execution.